### DISCUSSION TITLE
ComboboxButton(and some input fields) non-responsive in deeply nested component

### DISCUSSION THREAD

**Discussion by dulanovic1304** [tailwindlabs/headlessui#3132] [Help]
**What package within Headless UI are you using?**

@headlessui/vue

**What version of that package are you using?**

v1.7.20

**What browser are you using?**

Chrome, Mozilla Firefox, Brave

**Describe your issue**

A custom-made VueJS component which utilizes Combobox API for dynamic search(and select) is not responsive when loaded in 1 of 3 scenarios. It is deeply nested component, 5th layer, 1st one being loaded from blade template. Other 2 usages of the same component are functional, layer or two higher in our component hierarchy.
There is no Console output whatsoever, and the AJAX/axios search initiated by the component in question works correctly. Problem seems to be with non-responsive ComboboxButton, which does not open nor close the Combobox, and ComboboxOptions not being selectable by the mouseclick event. Only solution was to downgrade package version to the previous one (1.7.19).

**Comment by RobinMalfait** [comment]
Hey!

Can you share a minimal reproduction repo so that we can take a look?

**Reply by dulanovic1304** [reply]
[Repro link](https://github.com/dulanovic1304/headlessui-vuejs-v1-7-20-bug-repro)

Sorry for the delay!
